src/errors/test-run/templates.js

```diff
@@ -39,6 +39,12 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.couldNotOpenPage]: err => markup(err, `
+        Could not open a webpage because of the following error: <code>${err.errMsg}</code>
```

Comment:
Error message shouldn't be wrapped in `<code>` (see [hammerhead errors](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/messages.js)). And I think we don't need any prefix - we can pass hammerhead error as is.

```